### PR TITLE
PMML: Add more dev-check tests and improve the independent array regex parsing

### PR DESCRIPTION
**Pull request by kaknikhil**
This PR does two things
1. Adds a few more dev-check pmml tests 
2. Improves the independent variable parsing to account for spaces in the expression

See individual commits for details

### CODE REVIEW

File: src/ports/postgres/modules/pmml/test/pmml_ordinal.sql_in [hunk 1]
```diff
@@ -208,44 +215,63 @@ INSERT INTO "Multinom_test"("FEAT1", feat2, cat, g) VALUES
...
+-- probit
 drop table if exists ordinal_probit, ordinal_probit_summary;
 SELECT ordinal(
-    '"Multinom_test"',
+    '"Ordinal_test"',
     'ordinal_probit',
     'cat',
     'ARRAY["FEAT1", feat2]',
-    '0<1<2',
+    '2<0<1',
```
Comment (khannaekta) [inline src/ports/postgres/modules/pmml/test/pmml_ordinal.sql_in:246]: why did we change this order?

File: src/ports/postgres/modules/pmml/test/pmml_rf.sql_in [hunk 2]
```diff
@@ -29,11 +36,11 @@ DROP TABLE IF EXISTS train_output, train_output_summary, train_output_group;
 SELECT forest_train('dt_golf'::text,         -- source table
                          'train_output'::text,    -- output model table
                          'id'::text,              -- id column
-                         'temperature::double precision'::text,           -- response
+                         'temperature'::text,           -- response
                          'humidity, windy'::text,   -- features
                          NULL::text,        -- exclude columns
                          'class'::text,      -- grouping
-                         5::integer,     -- num_trees
+                         3::integer,     -- num_trees
```
Comment (khannaekta) [inline src/ports/postgres/modules/pmml/test/pmml_rf.sql_in:43]: was this update for speed up ?

**Review by khannaekta — APPROVED** [review]
Thanks @kaknikhil for adding more test coverage. LGTM!
just have some minor questions. Feel free to merge once they are addressed.

File: src/ports/postgres/modules/pmml/test/pmml_ordinal.sql_in (hunk 1 above)
Comment (kaknikhil) [reply src/ports/postgres/modules/pmml/test/pmml_ordinal.sql_in:246]: I added a new order to test "2<0<1". The original "0<1<2" still exists in the `pmml_ordinal.sql_in` file. The diff tool is a bit confusing

File: src/ports/postgres/modules/pmml/test/pmml_rf.sql_in (hunk 2 above)
Comment (kaknikhil) [reply src/ports/postgres/modules/pmml/test/pmml_rf.sql_in:43]: Yes

**Pull request merged**